utils/rpms/daos.spec

```diff
@@ -393,6 +393,7 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 %{_bindir}/daos_engine
 %{_bindir}/daos_metrics
 %{_bindir}/ddb
+%{_libdir}/libddb.so
```

Comment:
Should put the library in daos_srv

Reply:
There should probably also be a change log unless we are deferring that until we land the feature?

Reply:
Ya, we're waiting until the feature lands before updating the change log to avoid constant conflicts every time masters merged into it.